run_tests.sh

```diff
@@ -111,6 +111,7 @@ declare -a defences=("tests/defences/test_adversarial_trainer.py" \
                      "tests/defences/test_pixel_defend.py" \
                      "tests/defences/test_reverse_sigmoid.py" \
                      "tests/defences/test_rounded.py" \
+                     "tests/defences/test_strip.py" \
```

Comment:
Could you please convert these tests to the new pytest pattern: https://github.com/Trusted-AI/adversarial-robustness-toolbox/wiki/ART-Unit-Testing

Reply:
Done. Please check that it's truly framework-independent. I'm not sure if it's only testing TF.